src/components/VerticalListMenu/examples/1.basic.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import VerticalListMenuStateless from '../VerticalListMenu';
+import { buildStatefulComponent } from '../../../util/state-management';
+
+const VerticalListMenu = buildStatefulComponent(VerticalListMenuStateless);
+
+export default React.createClass({
+	render() {
+		return (
+			<VerticalListMenu style={{ width: '250px' }}>
```

Comment:
I feel like we've talked about this before. My vote is for a string with `px` since it's more explicit but I don't have strong feelings.